manifests/init.pp

```diff
@@ -151,6 +160,7 @@
   Optional[Hdm::Gitdata]         $git_data               = undef,
   Optional[Hdm::Ldap_settings]   $ldap_settings          = undef,
   Optional[Sensitive[String[1]]] $ldap_bind_dn_password  = undef,
+  Optional[Hash[String,Enum['yaml', 'eyaml']]]  $custom_lookup_function = undef,
```

Comment:
Can you please update the default to empty hash?
```suggestion
  Optional[Hash[String,Enum['yaml', 'eyaml']]]  $custom_lookup_function = {},
```

From https://voxpupuli.org/docs/reviewing_pr/ :
> Are there new parameters with datatype Hash or Array? If possible, they should default to empty Hash or Array instead of undef. You can also enforce the datastructure like Array[String[1]] (that’s recommended if possible)

Reply:
Done